__tests__/app.test.js

```diff
@@ -309,3 +310,30 @@ describe("DELETE /api/comments/:comment_id", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  it("responds with a 200 status message", () => {
+    return request(app).get("/api/users").expect(200);
+  });
+  it("responds with an array of users", () => {
+    return request(app)
+      .get("/api/users")
+      .then((data) => {
+        expect(Array.isArray(data.body)).toBe(true);
+      });
+  });
+  it("responds with an array of users which contains a username, name and avatar URL", () => {
+    return request(app)
+      .get("/api/users")
+      .then(({ body }) => {
+        expect(body.length).toBe(4);
+        body.forEach((user) => {
```

Comment:
Users should arrive on a key of 'users' in an object, not directly on the 'body'. Please refactor this.